tests/JSONToPuzJSON.test.ts

```diff
@@ -1,6 +1,5 @@
 import { readFileSync, writeFileSync } from "fs"
-import { xdToJSON } from ".."
-import { puzToXD } from "../dist"
+import { puzToXD, xdToJSON } from ".."
```

Comment:
These references to dist seemed wrong so I updated them, but did they serve a purpose?

Reply:
likely auto-import